stable/aws-vpc-cni/README.md

```diff
@@ -20,13 +20,7 @@ To install the chart with the release name `aws-vpc-cni` and default configurati
 $ helm install --name aws-vpc-cni --namespace kube-system eks/aws-vpc-cni
 ```
 
-To install into an EKS cluster where the CNI is already installed, you can run:
-
-```shell
-helm upgrade --install --recreate-pods --force aws-vpc-cni --namespace kube-system eks/aws-vpc-cni
-```
-
-If you receive an error similar to `Error: release aws-vpc-cni failed: <resource> "aws-node" already exists`, simply rerun the above command.
+To install into an EKS cluster where the CNI is already installed, see [this section below](##-Adopting-the-existing-aws-node-resources-in-an-EKS-cluster)
```

Comment:
The Anchor link is not working for me in GitHub's Markdown viewer. Unless I'm missing something, the `#-` shouldn't be there, it should be `#Adopting-the-existing-aws-node-resources-in-an-EKS-cluster`